group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupMetadataManager.java

```diff
@@ -1670,7 +1699,7 @@ private static void throwIfStreamsGroupMemberEpochIsInvalid(
             // If the member comes with the previous epoch and has a subset of the current assignment partitions,
             // we accept it because the response with the bumped epoch may have been lost.
             if (receivedMemberEpoch != member.previousMemberEpoch()
-                || !areOwnedTasksContainedInAssignedTasks(ownedActiveTasks, member.assignedTasks().activeTasks())
```

Comment:
Is `areOwnedTasksContainedInAssignedTasks` still used or could we remove it?

Reply:
Yes, it's still used for standby tasks.